### DISCUSSION TITLE
Resolution of output

### DISCUSSION THREAD

**Discussion by jardaoravec** [ultralytics/yolov5#2458] [General]
Hi please can you show me how to change webcam resolution output ? Can I change this resolution? 

img from dataloader is  (1, 3, 480, 640)  shape in line 68 detect.py, I need to change to (1, 3, 448, 640) 
Can you help me ?

**Comment by glenn-jocher** [comment]
@jardaoravec `python detect.py --img-size IMG-SIZE` defines the inference size for the longer dimension (width or height) of the batch. Short side is automatically adjusted per model stride constraints for minimum GFLOPS and is not user definable.

**Reply by glenn-jocher** [reply]
https://github.com/ultralytics/yolov5/blob/747c2653eecfb870b1ed40b1e00e0ef209b036e9/detect.py#L151
